Source/WebCore/rendering/style/GridTrackSize.h

```diff
@@ -128,9 +142,9 @@ class GridTrackSize {
 
 private:
     GridTrackSizeType m_type;
-    GridLength m_minTrackBreadth;
-    GridLength m_maxTrackBreadth;
-    GridLength m_fitContentTrackBreadth;
+    Style::GridTrackBreadth m_minTrackBreadth;
+    Style::GridTrackBreadth m_maxTrackBreadth;
+    Style::GridTrackBreadth m_fitContentTrackBreadth;
```

Comment:
Consider a default value?
```suggestion
    Style::GridTrackBreadth m_fitContentTrackBreadth { 0_css_px };
```